docs/test/perfTestPlan.adoc

```diff
@@ -0,0 +1,327 @@
+= Alcor Performance Test Plan
+Chun-Jen Chung <cchung@futurewei.com>, Liguang Xie <lxie@futurewei.com>
+v0.1, 2020-07-22; v0.2, 2021-06-15
+:toc: right
+:imagesdir: images
+
+== Overview
+
+In this document, we outline the performance test plan for Alcor SDN Control Plane and the performance comparison with
+OpenStack Neutron - the de facto networking control plane for OpenStack-based cloud.
+This test plan details test environment, tools, system monitoring and most importantly, test cases against Alcor and Neutron.
+The test cases includes a set of established test cases offered by Rally, which is an open-source performance test framework
+and benchmarking tool, as well as new test cases customized for public cloud scenarios.
+
+To ensure fair comparison between Alcor and Neutron, our setup proposal is as follows:
+
+- Same set of controller nodes will be used for Neutron control plane and Alcor control plane.
+When one control plane is under test, the other will be completely turned off.
+- Same set of compute nodes will be used to host VMs and/or other customer workloads.
+- Same benchmarking tool and test cases will be used to conduct the performance test.
+- Same set of performance metrics to measure the performance and scalability of a control plane.
+
+This test plan covers an ongoing effort to cover the most important cloud operations in terms of networking performance,
+starting with network/VPC, subnet and ports.
+We will continue to extend the coverage including routers, security groups and other networking objects.
+
+== Test Environment
+
+This section includes preparation and details of our test environment including hardware specification of servers,
+network parameters, operation system and perf test tools.
+
+=== Overall Setup
+
+Our performance test environment contains 3 types of nodes:
+
+- Controller nodes: hosting both OpenStack control plane, Alcor control plane and ElasticSearch analytics cluster
```

Comment:
kubernetes cluster for Alcor control plane, ElasticSearch analytics and Jaeger tracing cluster

Reply:
Fixed.